pkgs/applications/office/ticktick/default.nix

```diff
@@ -0,0 +1,79 @@
+{ lib
+, fetchurl
+, stdenv
+, wrapGAppsHook
+, dpkg
+, autoPatchelfHook
+, glibc
+, gcc-unwrapped
+, nss
+, libdrm
+, mesa
+, alsa-lib
+, xdg-utils
+, systemd
+}:
+stdenv.mkDerivation (final: {
+  pname = "ticktick";
+  version = "1.0.80";
+
+  src = fetchurl {
+    url = "https://d2atcrkye2ik4e.cloudfront.net/download/linux/linux_deb_x64/${final.pname}-${final.version}-amd64.deb";
+    sha256 = "1iyxn9rdv6p8d36swqpxb346arblk00yhv9z2qf6i6r2a4sbrbqh";
+  };
+
+  system = "x86_64-linux";
```

Comment:
why is this here?

Reply:
@eclairevoyant I mentioned earlier in the PR that I was following `skypeforlinux`'s example as another DEB repackage effort.

Reply:
I would say you can remove it then if there's no reason to have it, after all we have meta.platforms. Not every package is a good example sadly